js/app.js

```diff
@@ -211,5 +269,6 @@
    * Hint: Do not declare a new function expression as an event handler.
    */
 
+  btn15.addEventListener('click', setMyLightClass.bind(btn15));
```

Comment:
This is missing some parameters, naming who do you tell `setMyLightClass` what class to change to?